model/database/registry.yaml

```diff
@@ -127,238 +127,185 @@ groups:
+              brief: "[InterSystems Caché](https://www.intersystems.com/products/cache/)"
+              stability: development
             - id: cassandra
-              value: "cassandra"
-              brief: "Apache Cassandra"
-              stability: experimental
+              value: "cassandra"  # trademark on 'cassandra'
+              brief: "[Apache Cassandra](https://cassandra.apache.org/)"
+              stability: development
             - id: clickhouse
-              value: "clickhouse"
-              brief: "ClickHouse"
-              stability: experimental
-            - id: cloudscape
-              value: "cloudscape"
-              deprecated: "Replaced by `other_sql`."
-              brief: "Deprecated, use `other_sql` instead."
-              stability: experimental
+              value: "clickhouse"  # no wordmark
+              brief: "[ClickHouse](https://clickhouse.com/)"
+              stability: development
             - id: cockroachdb
-              value: "cockroachdb"
-              brief: "CockroachDB"
-              stability: experimental
-            - id: coldfusion
-              value: "coldfusion"
-              deprecated: "Removed."
-              brief: "Deprecated, no replacement at this time."
-              stability: experimental
-            - id: cosmosdb
-              value: "cosmosdb"
-              brief: "Microsoft Azure Cosmos DB"
-              stability: experimental
+              value: "cockroachdb"  # trademark on 'cockroachdb'
+              brief: "[CockroachDB](https://www.cockroachlabs.com/)"
+              stability: development
             - id: couchbase
-              value: "couchbase"
-              brief: "Couchbase"
-              stability: experimental
+              value: "couchbase"  # trademark on 'couchbase'
+              brief: "[Couchbase](https://www.couchbase.com/)"
+              stability: development
             - id: couchdb
-              value: "couchdb"
-              brief: "CouchDB"
-              stability: experimental
-            - id: db2
-              value: "db2"
-              brief: "IBM Db2"
-              stability: experimental
+              value: "couchdb"  # trademark on 'couchdb'
+              brief: "[Apache CouchDB](https://couchdb.apache.org/)"
+              stability: development
             - id: derby
-              value: "derby"
-              brief: "Apache Derby"
-              stability: experimental
-            - id: dynamodb
-              value: "dynamodb"
-              brief: "Amazon DynamoDB"
-              stability: experimental
+              value: "derby"  # trademark on 'derby'
+              brief: "[Apache Derby](https://db.apache.org/derby/)"
+              stability: development
             - id: edb
-              value: "edb"
-              brief: "EnterpriseDB"
-              stability: experimental
+              value: "edb"  # trademark on 'enterprisedb' and 'edb'
```

Comment:
it looks like enterprisedb is based on postgresql, so probably clients won't know the difference and could remove